doc/api/cli.markdown

```diff
@@ -0,0 +1,109 @@
+# Command Line Options
+
+<!--type=misc-->
+
+Node.js come with a wide variety of CLI options with multiple ways to execute scripts, built-in debugging, and a handful of other helpers.
+
+To view this documentation as a manual page in your terminal, run `man node`.
+
+## Synopsis
+
+`node [options] [v8 options] [script.js | -e "script"] [arguments]`
+
+`node debug [script.js | -e "script" | <host>:<port>] …`
+
+`node --v8-options`
+
+Execute without arguments to start the [REPL][].
+
+_For more info about `node debug`, please see the [debugger][] documentation._
+
+## Options
+
+### `-v`, `--version`
+Print node's version.
+
+### `-h`, `--help`
+Print node command line options.
+The output of this option is less detailed than this document.
+
+### `-e`, `--eval "script"`
+Evaluate the following argument as JavaScript.
+
+### `-p`, `--print "script"`
+Identical to `-e` but prints the result.
+
+### `-c`, `--check`
+Syntax check the script without executing.
+
+### `-i`, `--interactive`
+Opens the REPL even if stdin does not appear to be a terminal.
+
+### `-r`, `--require module`
+Preload the specified module at startup.
+
+Follows `require()`'s module resolution
+rules. module may be either a path to a file, or a node module name.
```

Comment:
`Node.js module name` sounds like a core module to me.

Would `--require <module>` work?